modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -732,8 +732,9 @@ void cv::gimpl::ie::GIEExecutable::run(cv::gimpl::GIslandExecutable::IInput  &in
 
     if (cv::util::holds_alternative<cv::gimpl::EndOfStream>(in_msg))
     {
-        // (3) Wait until all passed task are done.
+        GAPI_LOG_INFO(nullptr, "Got EndOfStream - wait until all passed task are done");
         m_reqPool->waitAll();
+        GAPI_LOG_INFO(nullptr, "all done");
```

Comment:
so would it spam console by default here?

Reply:
As i supposed EOS is a rare message (isn't it?) and must have been faced only once. 
Are my assumptions correct? If yes - no spam here, only countable output

Reply:
EOS may happen multiple times especially in start/stop scenarios.